### ISSUE TITLE
faux vs true italic

### ISSUE THREAD

**Issue by xinemata** [Kcaden20/critical-code-cookbook#12]
When I set a paragraph text to italic on Kirby, does it actually uses the `--fbItalic` variable (Tiresias PC Font Italic) or is it creating a faux italic using `--fBody` (Tiresias PC Font)? 

The [Gendered Cable Manifesto](https://criticalcode.recipes/contributions/the-gendered-cable-manifesto) is a compilation of quotes, where I see the possibility of legibility issues arising if it's defaulting to faux italic.

**Comment by Kcaden20** [comment]
Will take a look—this is the first time I’ve had to set a custom italic
typeface so I’ll test to make sure it’s working appropriately.

**Comment by xinemata** [comment]
Sounds good. Also this [article](https://spaceninja.com/2010/11/29/font-face-faux-styles/) might help.

**Comment by Kcaden20** [comment]
Great, thanks!

**Comment by Kcaden20** [comment]
This has been fixed—It took a second to find a way to distinguish FI from TI but in looking at the way the S&T interact when properly loaded (the S&T should be touching), I was able to confirm it is now loading in properly. 

See screenshot below documenting my finding. Left is Faux Italic, Right is True Italic, the Fontbook reference is pulled up to show the S&T touching.  
![Screen Shot 2022-06-23 at 2 18 58 PM](https://user-images.githubusercontent.com/13590895/175367689-6e339b28-aeb9-4de6-b8be-f00d2c0f2c7d.png)
